### ISSUE TITLE
Give player funds from group?

### ISSUE THREAD

**Issue by zochgcom** [sentanos/roblox-js#105]
Is there any way to give a player funds from a group?

**Comment by Neztore** [comment]
No.
Roblox-js is also no longer maintained.

**Comment by zochgcom** [comment]
aww

**Comment by suufi** [comment]
Noblox.js is maintained however, but is currently being rewritten.
https://GitHub.com/suufi/noblox.js

**Comment by EraGamer** [comment]
I suggest using Noblox too since it's maintained and updated. It also supports group payouts!

**Comment by sentanos** [comment]
Uhh there is a way... rbx.groupPayout. The wiki has more information about it

**Comment by ghost** [comment]
Roblox does not allow this. And you will most likely get banned.

**Comment by skriken** [comment]
Uhm what the hell are you talking about

**Comment by EraGamer** [comment]
Roblox does allow this as long as it's not malicious and is used at your own risk.

**Comment by ghost** [comment]
No. If your game does get popular. Roblox will see this and ban you for "giving away robux." I already asked a question about this here and you will get banned.

**Comment by EraGamer** [comment]
Roblox does not allow the advertising of "free Robux" or the selling of Robux but will not punish giving away Robux via a bot. 

Example: I have a discord bot linked to my account that I use for giving away Robux from groups I own. I generally use it to pay people who do work for me.

**Comment by ghost** [comment]
Thats fine. What im saying is that you can not give away robux for free. I asked on the dev-fourms and everyone said no you could not.

**Comment by skriken** [comment]
Link to thread so we know that those weren't retards.
